packages/test-utils/src/wrapper.js

```diff
@@ -421,7 +421,14 @@ export default class Wrapper implements BaseWrapper {
 
     Object.keys(data).forEach((key) => {
       // $FlowIgnore : Problem with possibly null this.vm
-      this.vm.$set(this.vm, [key], data[key])
+      if (typeof data[key] === typeof {} && data[key] !== null) {
```

Comment:
Why are you using typeof `data[key] === typeof {}`, not `data[key] === 'object'` ?

Reply:
The null check was necessary because `null` is of type object. If you're updating an object with `{message: null}`, it will pass this conditional, which it technically shouldn't since it's not a nested object.